packages/cli/src/__tests__/commands/Generate.test.ts

```diff
@@ -147,4 +136,12 @@ function cleanSnapshot(str: string): string {
     .replace(/\d+ms/g, 'XXms')
     .replace(/\d+s/g, 'XXs')
     .replace(/\(version:.+\)/g, '(version: 0.0.0)')
+    .replace(new RegExp(getClientEngineType(), 'g'), 'TEST_ENGINE_TYPE')
+}
+
+function replaceEngineType(message: string | Error) {
+  if (message instanceof Error) {
+    throw message
+  }
```

Comment:
Why a throw here? 

I expected only a replace 🤔

Reply:
`Generate.parse` returns `Promise<string | Error>` and I needed to handle it somehow. I can do something like

```ts
if (message instanceof Error) {
  message = message.stack
}
```

here instead. I think it will be closer to the previous behavior, too (i.e., when `string | Error` was unconditionally matched with the snapshot).

Reply:
Oh I see 👍🏼 

But we don't really need to handle it since where `replaceEngineType` it's never an error?

I'm ok keeping it if you think it's useful.

Reply:
> But we don't really need to handle it

Well, yeah, I didn't think about it, but we can make this function return `string | Error` too and just return the error here as is.